src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlLinkFactory.java

```diff
@@ -199,9 +199,11 @@ protected Content getTypeParameterLinks(LinkInfo linkInfo) {
     protected Content getTypeParameterLink(LinkInfo linkInfo, TypeMirror typeParam) {
         HtmlLinkInfo typeLinkInfo = new HtmlLinkInfo(m_writer.configuration,
                 ((HtmlLinkInfo) linkInfo).getContext(), typeParam);
-        typeLinkInfo.excludeTypeBounds = linkInfo.excludeTypeBounds;
-        typeLinkInfo.excludeTypeParameterLinks = linkInfo.excludeTypeParameterLinks;
+        typeLinkInfo.showTypeBounds = linkInfo.showTypeBounds;
+        typeLinkInfo.linkTypeParameters = linkInfo.linkTypeParameters;
         typeLinkInfo.linkToSelf = linkInfo.linkToSelf;
+        typeLinkInfo.addLineBreaksInTypeParameters = linkInfo.addLineBreaksInTypeParameters;
+        typeLinkInfo.showTypeParameterAnnotations = linkInfo.showTypeParameterAnnotations;
         return getLink(typeLinkInfo);
```

Comment:
It seems like there out to be a more convenient constructor or "copy"-ish sort of method on `HtmlLinkInfo`.